Kconfig

```diff
@@ -649,14 +649,14 @@ comment "Debug SYSLOG Output Controls"
 
 config DEBUG_ERROR
```

Comment:
but why not remove DEBUG_ERROR, DEBUG_WARN and DEBUG_INFO directly?

Reply:
I just don't want to disrupt too much with original code and there are many places in the code there only CONFIG_DEBUG_ERROR/WARN/INFO is used.

Reply:
Ok